specifyweb/frontend/js_src/lib/components/Merging/CompareSubView.tsx

```diff
@@ -147,6 +147,10 @@ function MergeDialog({
     ...[mergedRecords, ...children].map((children) => children.length)
   );
 
+  const childrenLength = resources
+    .map((record) => f.maybe(record, getChildren))
```

Comment:
why are you getting children here again if the code already did that just a few lines up?
```
  const [children, setChildren] = useChildren(
    mergedRecords,
    getChildren,
    resources,
    relationship
  );
```

Reply:
not the same